src/config.js

```diff
@@ -1,6 +1,6 @@
 const configs = [
   {
-    DOMAINS: ["localhost", "10.0.0.107", "sundai.fritz.box", /192\.168\..*/, /.*\.ngrok\.io/],
+    DOMAINS: ["localhost", "151.217.248.171", "10.0.0.107", "sundai.fritz.box", /192\.168\..*/, /.*\.ngrok\.io/],
```

Comment:
Do we need new domain here?

Reply:
it's my address at the camp. Actually it's accessible from anywhere in the world 😆 

Can we leave it, as it's more convenient to me?

Reply:
If you wish :)